### PR TITLE
Fix 0.7 depwarns in function signatures

### PR DESCRIPTION
**Pull request by mforets**
Fixes some (but not all) deprecation warnings in v.0.7.

### CODE REVIEW

**Comment by mforets** [comment]
ping?

**Comment by acroy** [comment]
Thanks! It seems that the way the package is initialized for Travis has to be changed under v0.7?

**Comment by acroy** [comment]
I think this still doesn't resolve the issue with the testing. For other packages I saw that they basically removed the `script` part altogether in `travis.yml`?

**Comment by acroy** [comment]
@mforets I changed the Travis script. Could you please rebase this PR against master? Then we can directly see which routines are still causing trouble ...

File: test/runtests.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 using Expokit
-using Base.Test
+using Compat.Test
```
[the remark this answers is not in the document]
Comment (mforets) [reply test/runtests.jl]: please go ahead if you want to try this by yourself now, so we are done with porting to 0.7

**Comment by mforets** [comment]
@acroy  thanks for the update. now i get

```julia
julia> using Expokit
[ Info: Recompiling stale cache file /Users/mforets/.julia/compiled/v0.7/Expokit/vT6yJ.ji for Expokit [a1e7a1ef-7a5d-5822-a38c-be74e1bb89f4]
┌ Warning: Package Expokit does not have LinearAlgebra in its dependencies:
│ - If you have Expokit checked out for development and have
│   added LinearAlgebra as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with Expokit
└ Loading LinearAlgebra into Expokit from project dependency, future warnings for Expokit are suppressed.
┌ Warning: Deprecated syntax `multiple line breaks between doc string and object` at /Users/mforets/.julia/dev/Expokit/src/phimv.jl:35.
│ Use `at most one line break` instead.
└ @ ~/.julia/dev/Expokit/src/phimv.jl:35
```

1) it complains that `LinearAlgebra` is missing from REQUIRE
2) some issue with line breaks in `phimv` line 35
3) testing the package still gives a ton of warnings, i've opened #33

**Comment by acroy** [comment]
Thanks! I am working on it ...

**Pull request merged**